### PR TITLE
Fixed images 1.10

### PR DESCRIPTION
**Pull request by matzew**
updating the CSV template to point to v1.10 - instead of nightly and regenerate the actual CSV (`make csv`)

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -41,29 +41,29 @@ spec:
...
-    - description: Represents an installation of a particular version of Knative Eventing
-      displayName: Knative Eventing
-      kind: KnativeEventing
-      name: knativeeventings.operator.knative.dev
-      statusDescriptors:
-      - description: The version of Knative Eventing installed
-        displayName: Version
-        path: version
-      version: v1alpha1
+      - description: Represents an installation of a particular version of Knative Serving
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:44]: This diff is quite odd. When i run `make csv` I don't get the same diff :thinking:. Are we having different versions of `yq` or something?

File: olm-catalog/serverless-operator/csv.template.yaml [hunk 2]
```diff
@@ -338,7 +338,7 @@ spec:
               containers:
                 - name: knative-openshift-ingress
                   # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress
+                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.10.0:knative-openshift-ingress
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/csv.template.yaml:341]: Uhoh. I just realized that my script is wrong... These images are missing from the relatedImages block now :see_no_evil:

Comment (matzew) [reply olm-catalog/serverless-operator/csv.template.yaml:341]: :smile_cat: 
wanna PR something to master, and I cherry-pick :cherries: that here, @markusthoemmes ?

Comment (markusthoemmes) [reply olm-catalog/serverless-operator/csv.template.yaml:341]: yeah on my TODO for today.

**Comment by matzew** [comment]
/close

b/c of #527

**Pull request closed without merging**